src/zajecia10/zadanie1/Test.java

```diff
@@ -0,0 +1,38 @@
+package zajecia10.zadanie1;
+
+import java.util.Scanner;
+
+public class Test {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        Person person = null;
+        String firstName;
+        String lastName;
+        int age;
+        String pesel;
+
+        System.out.print("Podaj imie: ");
+        firstName = scanner.nextLine();
+        System.out.print("Podaj nazwisko: ");
+        lastName = scanner.nextLine();
+        System.out.print("Podaj wiek: ");
+        age = scanner.nextInt();
+        System.out.print("Podaj pesel: ");
+        pesel = scanner.next();
+
+        try {
+            person = new Person(firstName, lastName, age, pesel);
+        } catch (NameUndefinedException | IncorrectAgeException e) {
+            System.err.println(e.getMessage());
+        }
+
+        try {
```

Comment:
w taki sposob nie obslugujemy przypadków gdzie uzyta referencja moze byc nie zainicjalizowana ( czyli jest nullem). Jesli referencja jest nullem to jest powazny problem programistyczny i lepiej pozwolic programowi zakonczyc dzialanie. A pozniej to poprawic

Reply:
done